### ISSUE TITLE
TypeError: Cannot read property 'color' of undefined

### ISSUE THREAD

**Issue by JustinBeckwith** [IonicaBizau/github-stats#23]
Greetings!  Was trying to run this, and it threw a runtime error at me :( 

```
$ github-stats -u JustinBeckwith
info  Fetching data...
/Users/beckwith/.nvm/versions/node/v6.10.3/lib/node_modules/github-stats/node_modules/gh-polyglot/lib/index.js:122
                color: GitHubColors.get(cLang).color
                                              ^

TypeError: Cannot read property 'color' of undefined
    at /Users/beckwith/.nvm/versions/node/v6.10.3/lib/node_modules/github-stats/node_modules/gh-polyglot/lib/index.js:122:47
    at Array.map (native)
    at /Users/beckwith/.nvm/versions/node/v6.10.3/lib/node_modules/github-stats/node_modules/gh-polyglot/lib/index.js:118:42
    at /Users/beckwith/.nvm/versions/node/v6.10.3/lib/node_modules/github-stats/node_modules/gh-polyglot/lib/index.js:55:9
    at /Users/beckwith/.nvm/versions/node/v6.10.3/lib/node_modules/github-stats/node_modules/gh.js/lib/index.js:170:28
    at GitHub.checkResponse (/Users/beckwith/.nvm/versions/node/v6.10.3/lib/node_modules/github-stats/node_modules/gh.js/lib/index.js:122:42)
    at /Users/beckwith/.nvm/versions/node/v6.10.3/lib/node_modules/github-stats/node_modules/gh.js/lib/index.js:88:18
    at /Users/beckwith/.nvm/versions/node/v6.10.3/lib/node_modules/github-stats/node_modules/jsonrequest/lib/index.js:57:9
    at opt_callback (/Users/beckwith/.nvm/versions/node/v6.10.3/lib/node_modules/github-stats/node_modules/tinyreq/lib/index.js:53:9)
    at IncomingMessage.<anonymous> (/Users/beckwith/.nvm/versions/node/v6.10.3/lib/node_modules/github-stats/node_modules/tinyreq/lib/index.js:85:13)
```

Good luck!

**Comment by IonicaBizau** [comment]
TL;DR You have many languages. 😁
http://ionicabizau.github.io/github-profile-languages/?user=JustinBeckwith

The problem seems to come from `gh-polyglot`. Related: https://github.com/IonicaBizau/github-colors/issues/26

**Comment by IonicaBizau** [comment]
Fixed in https://github.com/IonicaBizau/node-gh-polyglot/pull/15

You will need to `npm i github-stats@latest`.
Updated the other dependencies too (fixes the legend on macOS).

**Comment by JustinBeckwith** [comment]
Much better!  Thank you so much for the snappy response.
